Readme.md

```diff
@@ -28,23 +28,43 @@ SimpleITK is licensed under the [Apache License](http://www.opensource.org/licen
 Development
 ===========
 
-SimpleITK uses the [Git](http://git-scm.com/) distributed version control system.  The main repository is hosted on [itk.org](https://itk.org/SimpleITK.git) and mirrored to [Github](https://blowekamp@github.com/SimpleITK/SimpleITK.git).  There are two main branches, master and next. The "master" is a stable branch of the code, and suitable for most users, while "next" contains future features that are unlikely to be stable.
-
-The SimpleITK [Wiki](https://itk.org/Wiki/SimpleITK) is the main site for all things SimpleITK. The site includes information on installing the toolkit's binary distributions, building the toolkit, user support documentation and much more. The API documentation is maintained using [Doxygen](https://www.itk.org/SimpleITKDoxygen/html/annotated.html).
-
+SimpleITK uses the [Git](http://git-scm.com/) distributed version control system.  The main repository is hosted on [itk.org](https://itk.org/SimpleITK.git) and mirrored to [Github](https://blowekamp@github.com/SimpleITK/SimpleITK.git).  
+
+Contribute
+--------
+We encourage any contributions from the community!
+
+To submit code to SimpleITK, simply follow the standard GitHub workflow. More information can be found [here](https://guides.github.com/activities/forking/) and [here](https://gist.github.com/Chaser324/ce0505fbed06b947d962) regarding the workflow and getting your computer setup.
+
+Branches
+* **next** - Contains future features that are unlikely to be stable. This is the default branch you should push changes to
```

Comment:
No branch/pull request should be based on the next branch. They must be based off the a commit in the master or next branch. The next branch is also used for extensive nightly regression testing.